screens/StudentInform2.tsx

```diff
@@ -3,18 +3,39 @@ import StyledButton from '../components/StyledButton';
 
 import List from '../components/List';
 import { HomeScreenProps } from './HomeScreen';
-import { useState } from 'react';
+import React, { useState } from 'react';
 import backendFetch from '../lib/backendFetch';
+import StyledAlternativeInput from '../components/StyledAlternativeInput';
 
 const image = require('../assets/images/background.png');
 
 export default function StudentInform2({
   navigation: { navigate },
 }: HomeScreenProps) {
-  const [qualities, setQualities] = useState([
-    { item: 'Example', key: 'Example' },
-  ]);
-  const [hobbies, setHobbies] = useState([{ item: 'Example', key: 'Example' }]);
+  const [qualities, setQualities] = useState('');
+  const [hobbies, setHobbies] = useState('');
+
+  function updateProfile() {
+    const missingFields: string[] = [];
+    console.log('state: ', { qualities, hobbies });
+    //replaceAll doesn't exist, so string.replace with this exact regex works fine
+    if (!qualities.replace(/,/g, '').trim()) {
+      missingFields.push('You have more qualities than this');
+    }
+    if (!hobbies.replace(/,/g, '').trim()) {
+      missingFields.push('Fill in at least 1 hobby');
+    }
```

Comment:
You can split a string into an array using the `.split()` method. I would write it as the following:
```suggestion
const qualitiesArr = qualities
      .split(",") // Splits the string into an array
      .map((q) => q.trim()); // Removes whitespace
      .filter((q) => q); // Removes empty strings
    const hobbiesArr = hobbies
      .split(",") // Splits the string into an array
      .map((h) => h.trim()); // Removes whitespace
      .filter((h) => h); // Removes empty strings
    if (qualitiesArr.length < 1) {
      missingFields.push("Fill in at least 1 quality");
    }
    if (hobbiesArr.length < 1) {
      missingFields.push("Fill in at least 1 hobby");
    }
```

has a bit of a rough variable naming scheme due to being unable to change the `qualities` and `hobbies` variable names but IMO is a lot better.